.github/workflows/go-test.yml

```diff
@@ -51,32 +49,27 @@ jobs:
 
     strategy:
       matrix:
-        flutter_version: ['3.3']
+        flutter_version: ["3.3", "3.7"]
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Setup Go
-        uses: actions/setup-go@v2
-        with:
-          go-version: '1.19'
-
-      - name: Cache Go modules
-        uses: actions/cache@v2
+        uses: actions/setup-go@v3
         with:
-          path: ~/go/pkg/mod
-          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-go-
+          go-version: ${{ matrix.go-version }}
+          cache: true
 
       - name: Build binary
-        run: go install .
+        run: |
+          go build
+          mv ./poe2arb /usr/local/bin
```

Comment:
Why did you change it like that?

Reply:
Because `go install .` doesn't work in Go 1.20 – the tests were failing with `poe2arb not found`.

I think it might be related to [this change](https://tip.golang.org/doc/go1.20#go-command):

> The directory $GOROOT/pkg no longer stores pre-compiled package archives for the standard library: go install no longer writes them, the go build no longer checks for them, and the Go distribution no longer ships them.

Reply:
No, it's not that. It must be something else. We'll figure it

Reply:
i'm gonna ask a fellow gopher

@shilangyu any ideas?